egov-indexer/fsm-inbox-indexer.yml

```
@@ -71,7 +71,7 @@ ServiceMaps:
           - inJsonPath: $.address.locality.code
             outJsonPath: $.Data.locality
           externalUriMapping:
-          - path: http://egov-workflow-v2.pucar-prod:8080/egov-workflow-v2/egov-wf/process/_search
+          - path: http://egov-workflow-v2.egov:8080/egov-workflow-v2/egov-wf/process/_search
```

yamllint (1.35.1)

[error] wrong indentation: expected 11 but found 10

(indentation)